src/wp-includes/class-wp-classic-to-block-menu-converter.php

```diff
@@ -34,7 +34,7 @@ public static function convert( $menu ) {
 		$menu_items = wp_get_nav_menu_items( $menu->term_id, array( 'update_post_term_cache' => false ) );
 
 		if ( empty( $menu_items ) ) {
-			return array();
+			return '';
```

Comment:
Is this a BC (backward compat) issue?

tl;dr I think no.

Why?

The method shipped in 6.3.0 is clearly documented as returning a `string`, not an `array`. Returning an `array` then is a bug that needs to be fixed.